server/run.sh

```
@@ -1,4 +1,12 @@
 source .venv/bin/activate
+# needed for autonomy setup
+if [ -f "/opt/ros/humble/setup.bash" ]; then
+    source /opt/ros/humble/setup.bash
```

Shellcheck (0.11.0)

[info] Not following: /opt/ros/humble/setup.bash was not specified as input (see shellcheck -x).

(SC1091)